deployment/terraform/storage.tf

```diff
@@ -1,3 +1,17 @@
+module "ecr_catalogs" {
+  source = "github.com/azavea/terraform-aws-ecr-repository?ref=1.0.0"
+
+  repository_name         = "noaa-flood-catalogs"
+  attach_lifecycle_policy = true
+}
```

Comment:
I set up an ECR repository for the `noaa-flood-catalogs` image as well. We typically pull that image down and run it locally, but I think it could be useful to just pull a prebuilt image.